server/src/db/seeders/AttractionSeeder.js

```diff
@@ -6,14 +6,17 @@ class AttractionSeeder {
       {
         name: "Baha'i Gardens",
         description: "A very beautiful garden by the ocean.",
+        locationId: 1
       },
       {
         name: "Jean's House",
-        description: "Home to million dollar app ideas and the world's greatest Overwatch player.",
+        description: "Home to million dollar apps and the world's greatest Overwatch player.",
```

Comment:
in the future, please avoid goofy seeds. While fun, this may be read over and be very confusing to an employer